_overviews/FAQ/index.md

```diff
@@ -120,7 +120,7 @@ with only minor supplements.  That's why `versionString` and
 `versionNumberString` report that Scala 2 is in use:
 
 ```
-Welcome to Scala 3.2.0 (17.0.3, Java OpenJDK 64-Bit Server VM).
+Welcome to Scala 3.3.0 (17.0.3, Java OpenJDK 64-Bit Server VM).
 Type in expressions for evaluation. Or try :help.
 
 scala> util.Properties.versionNumberString
```

Comment:
there's a line below this one saying `val res0: String = 2.13.11`.

Reply:
Good eye! But I didn't change it because that's what 3.3.0 actually reports, since 3.3.0 predates 2.13.12. Probably the upgrade will land in 3.3.2 and then we can bump it.

I'll add a little note about it, since this is far from obvious.

Reply:
Ha, actually the number is too high, not too low! Both 3.3.0 and 3.3.1 use 2.13.10 😢

Reply:
ah, ok. TIL. :)

Reply:
Note that I'm pursuing it over at https://github.com/lampepfl/dotty/pull/18525